src/pages/Applications/index.tsx

```diff
@@ -0,0 +1,9 @@
+import List from "./List";
+
+export default function Applications() {
```

Comment:
index files should only contain exports not whole components

Reply:
done